Command/DoctrineCommand.php

```diff
@@ -137,4 +156,12 @@ private static function injectContainerToMigrations(ContainerInterface $containe
             $migration->setContainer($container);
         }
     }
+
+    public static function getStringParameter(ContainerInterface $container, string $name): string
```

Comment:
```suggestion
    private static function getStringParameter(ContainerInterface $container, string $name): string
```

Reply:
:thinking: why would I do that?

Reply:
Because this is not used anywhere except this class.

Reply:
Oh sorry I read that backwards, fully agree!